src/schema/artwork/index.js

```diff
@@ -705,6 +706,70 @@ export const artworkFields = () => {
       type: GraphQLString,
       resolve: artwork => (isEmbeddedVideo(artwork) ? null : artwork.website),
     },
+    framed: {
+      type: ArtworkInfoRowType,
+      resolve: ({ framed }) => {
+        if (!framed) {
+          return null
+        }
+        return {
+          label: "Framed",
+          details: "",
```

Comment:
#minor. It's boolean, so maybe we can simply return `true` or `false`, if we want to differentiate unset value. However, it's a required field in CMS for BNMO.

Reply:
My understanding is that in Metaphysics we want to have a value that can be directly displayed in clients. So as far as clients are concerned - they either get a string to display or get null. The implementation as boolean in gravity is kinda masked here.

That is seems to be the direction we are going lately - just get metaphysics to return what clients will use without any additional checks. Let me know if I'm misunderstanding something here @starsirius.

Reply:
Yeah I agree with the response can be client-driven, and I simply wondered if this would be too specific to a single client, i.e. Force. Can be an ongoing discussion!